### DISCUSSION TITLE
Install on a subfolder

### DISCUSSION THREAD

**Discussion by dpicon83** [eusonlito/Password-Manager#30] [Q&A] [answered]
Hi, first off all, congrats for the awesome App. 

I would like to use it with my IT team in my University. But our server provides a subfolder to install it. In my tests, I couldn't run the project on a URL with subfolder, like https://test.localhost/password/
The first problem is that I received a 404 error if I don't put the index.php on the URL. If I force the index.php, the system redirects to a wrong URL for login, removing the subfolder from the URL, like https://test.localhost/user/auth

If I create a subdomain, like https://passwords.test.localhost it works perfectly, but I can't do it on my production server. 

On the .env file I put the right address, with the subfolder, on the APP_URL, but it seems like the system is ignoring that config. 

There ir some trick to make it work in this scenario? 

tks

**Comment by eusonlito** [comment, marked as answer]
Laravel by default does not support subfolder installation, but you can try this post https://learnwithdaniel.com/2021/03/run-laravel-project-in-subfolder-apache/

**Reply by dpicon83** [reply]
Yeah, I alredy tryed that before writing, but it didn't work, I thought it's because the redirect of the authentication proccess. But that's OK, I'll try to figure out how to do it and, if I have success, I post it here. 


thanks anyway

**Reply by eusonlito** [reply]
After change the `APP_URL` do you cleared laravel cache? `php artisan optimize`

**Reply by dpicon83** [reply]
Yes, I did. The result was the same.
